apps/notepad/notepad.talon

```diff
@@ -0,0 +1,5 @@
+app: notepad
+-
+
+tag(): user.tabs
```

Comment:
We live in a frightening future :D

Reply:
Actually, is this conditional on Win11 notepad?

Reply:
Yes. You think I should remove it for that reason?

Reply:
I would just keep it personally, especially since Windows 10 is EOL in ~1 year anyway? It's usefulness when the feature exists probably outweighs the minor annoyance to have those commands in context when it doesn't? That said I don't use windows and it won't affect me, so take my opinion lightly.

Reply:
That was kind of my thinking. Most people will soon be on windows 11. And it's not like we don't have commands available in Talon in some context that doesn't work already.

Reply:
Yeah, I think that's fine, I just wanted to discuss it / have our thinking on the record.

If we had an easy `win10`/`win11` tag already available, it might be nice to conditionalize it (although, one extra file 🤢), but agree it's not worth it for this alone. I do think the idea in general is something we would be open to in the future though.